### PR TITLE
Update project

### PR DESCRIPTION
**Pull request by kunkuma**
The dependencies, gradle and other deprecated code have been updated to suit modern android phones.

### CODE REVIEW

**Comment by mdvmeijer** [comment]
Thx for the PR

File: app/build.gradle [hunk 1]
```diff
@@ -4,22 +4,20 @@ apply plugin: 'com.chaquo.python'
...
-        minSdkVersion 16
-        targetSdkVersion 30
+        minSdkVersion 27
+        targetSdkVersion 34
         versionCode 1
         versionName "1.0"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         ndk {
-            abiFilters "armeabi-v7a", "x86"
+            abiFilters "arm64-v8a"
```
Comment (mdvmeijer) [inline app/build.gradle]: Please add x86 to this so that it works with emulators running on x86 machines

Comment (kunkuma) [reply app/build.gradle]: Sure. The side effect though is that this creates a longer building time.

**Review by mdvmeijer — APPROVED** [review]
Looks good

**Pull request merged**